crates/proof-of-sql/src/base/database/union_util.rs

```diff
@@ -201,20 +201,21 @@ pub fn column_union<'a, S: Scalar>(
 }
 
 /// Union multiple tables with compatible schemas into a single table
-///
-/// # Panics
-/// This function should never panic as long as it is written correctly
+#[expect(clippy::missing_panics_doc)]
 pub fn table_union<'a, S: Scalar>(
     tables: &[Table<'a, S>],
     alloc: &'a Bump,
-    schema: Vec<ColumnField>,
 ) -> TableOperationResult<Table<'a, S>> {
```

Comment:
We shouldn't allow unioning a single table because otherwise the error `TableOperationError::UnionNotEnoughTables` will have a wrong description.